lib/smart_answer_flows/marriage-abroad/outcomes/outcome_opposite_sex_marriage_in_japan.govspeak.erb

```diff
@@ -55,6 +55,6 @@
              locals: { calculator: calculator } %>
 
 
-  You can pay by credit card, but not by personal cheque. The embassy in Tokyo will also accept cash.
+  <%= render partial: 'pay_by_credit_card_but_not_personal_cheque.govspeak.erb' %>
```

Comment:
@floehopper: I've named this partial `pay_in_japan` in https://github.com/alphagov/smart-answers/pull/2507/commits/0a13dab61a3d389c4615e829b10558f8d13711ba.